collector/settings.mdx

```diff
@@ -335,7 +335,7 @@ Note that the `aws_endpoint_*` settings are only relevant if you are using custo
     </tr>
     <tr>
       <td>db_use_iam_auth (<code>DB_USE_IAM_AUTH</code>)</td>
-      <td>[none]</td>
+      <td>false</td>
```

Comment:
Side note (maybe for the future), but we should make up our mind about how to say "there is no default value".

We use `[none]` or `n/a` here and there, and for the case like this, while "[none]" is correct (in the way that the golang treats that as boolean "false") but it's not so clear from here.

Reply:
Ah, my bad for not making this clearer. I originally started using `[none]` as "this has no default but is ignored if you don't set it" and `n/a` as "this has no default, it must be configured (for a certain hosting provider or hosting configuration)".

This wasn't a clear delineation to begin with, because arguably `db_sslrootcert_contents` in the first commit should have been `[none]` rather than `n/a`, and I'm sure over time we've added more settings that don't follow this pattern (since I didn't make it clear), but that was the idea. I think your suggested change here makes sense, though.

Reply:
I see, thanks for explaining. Yeah, I think it'd be good to note it (can even live in this docs page too), and enforce the usage.
Let me see if I can make a PR for this 👍